src/main/java/com/throughapinhole/photoportfolio/utils/Utils.java

```diff
@@ -6,6 +6,11 @@
 final public class Utils {
     /** Value for a Category which has no parent */
     final public static int CATEGORY_NO_PARENT = -1; 
+    public enum EntityType {
```

Comment:
why is this not final?

Reply:
I got a linting error about final not being applicable to type enum. I think by default enum values are always final.

Reply:
`Illegal modifier for the member enum EntityType; only public, protected, private & static are permitted`

Reply:
I will make it static though

Reply:
ah I see